tests/Unit/Utilities/Test_StdArrayHelpers.cpp

```diff
@@ -151,3 +152,31 @@ SPECTRE_TEST_CASE("Unit.Utilities.StdArrayHelpers.map_array",
         return x;
       }) == std::array<int, 1>{{4}});
 }
+
+namespace {
+DEFINE_ARRAY_BINOP(int, int, int, operator+, std::plus<>())
+DEFINE_ARRAY_BINOP(double, double, double, f, std::multiplies<>())
+
+DEFINE_ARRAY_INPLACE_BINOP(int, int, operator+=, std::plus<>())
+DEFINE_ARRAY_INPLACE_BINOP(double, double, g, std::multiplies<>())
+
+SPECTRE_TEST_CASE("Unit.Utilities.StdArrayHelpers.define_array_binop",
+                  "[Utilities][Unit]") {
+  CHECK(std::array<int, 2>{{1, 2}} + std::array<int, 2>{{2, 3}} ==
+        std::array<int, 2>{{3, 5}});
+  CHECK(
+      f(std::array<double, 2>{{1.0, 2.0}}, std::array<double, 2>{{2.0, 3.0}}) ==
+      std::array<double, 2>{{2.0, 6.0}});
+}
+
+SPECTRE_TEST_CASE("Unit.Utilities.StdArrayHelpers.define_array_inplace_binop",
```

Comment:
Oh, sorry I missed this. Could you combine the TEST_CASEs into one? There's a decent overhead (1.2ish seconds) on supercomputers to start tests and having fewer cases will speed things up.

Reply:
geez, that seems like a lot for the number of tests we've got. 
These two tests combined.